sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -501,16 +501,13 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
 
         Seq(("true", (x: Long) => x == 0), ("false", (x: Long) => x > 0)).map { case (push, func) =>
```

Comment:
(BTW, let's don't forget to change `map` to `foreach` for `Filter applied on merged Parquet schema with new column should work` too if we are fine)